Toggl.Daneel/Extensions/Reactive/UITextFieldExtensions.cs

```diff
@@ -21,5 +23,25 @@ public static IObservable<string> Text(this IReactive<UITextField> reactive)
 
         public static Action<string> TextObserver(this IReactive<UITextField> reactive)
            => text => reactive.Base.Text = text;
+
+        public static IObservable<Unit> ShouldReturn(this IReactive<UITextField> reactive, bool shouldResignFirstResponder = true)
+        {
+            return Observable.Create<Unit>(observer =>
+            {
+                reactive.Base.ShouldReturn = textField =>
```

Comment:
should we keep the traditional delegates work by 
```suggestion
                reactive.Base.ShouldReturn += textField =>
```